docs/B-Computations/style-guidelines.md

```diff
@@ -252,17 +252,17 @@ For example:
 ```c
 const double PI = 3.14159;
 
- int main(void)
- {
-         double radius, area;
+int main(void)
+{
+        double radius, area;
 
-         printf("Enter radius : ");
-         scanf("%lf", &radius);
-         area = PI * radius * radius;
-         printf("The area of your circle is : %lf\n", area); 
+        printf("Enter radius : ");
+        scanf("%lf", &radius);
+        area = PI * radius * radius;
+        printf("The area of your circle is : %lf\n", area); 
 
-         return 0;
- }
+        return 0;
+}
 ```
 
 ### Macro Directive
```

Comment:
Typo for "occurence" on line 278. Should be "occurrence"

Also, add back ticks for `#define ` on the same line, as it refers to a code.

Reply:
Fixed in my latest commit! Thank you for the feedback!